### PR TITLE
Implement 'samply setup' to codesign binary

### PR DESCRIPTION
**Pull request by vvuk**

### CODE REVIEW

File: samply/src/mac/process_launcher.rs [hunk 1]
```diff
@@ -301,7 +305,7 @@ impl ExistingProcessRunner {
         };
 
         // always root pid first
-        queue_pid(pid, true);
+        queue_pid(pid, false);
```
Comment (mstange) [inline samply/src/mac/process_launcher.rs:308]: Seems unrelated, is it intentional?

**Review by mstange — APPROVED** [review]
Cool! Oh and please fix the typo in the commit message too. :)

File: samply/src/mac/process_launcher.rs (hunk 1 above)
Comment (vvuk) [reply samply/src/mac/process_launcher.rs:308]: Unrelated yeah, but also intentional; happened to catch it while looking at this and accidentally included it in here. It should be a failure if we can't attach to the root pid, because the profile will be empty.
